step-functions/nino_issue_credential.asl.json

```diff
@@ -333,11 +365,24 @@
           "BackoffRate": 2
         }
       ],
+      "Next": "Fetch Current Time For VC Issued Event",
+      "ResultSelector": {
+        "value.$": "$.Payload"
+      },
+      "ResultPath": "$.vcClaimSetSignature"
+    },
+    "Fetch Current Time For VC Issued Event": {
```

Comment:
Similar to my previous comment, in this instance, you are sending the TxMA events both at the same time. Any reason it needs 2 different timestamps? I think we can leave it as 1 lambda to get the time and 1 `EventBridge: PutEvents` to send all the TxMA events.

Reply:
I don't know how important precision of these events are. When looking at the timestamps after doing a run through, they are different but only by milliseconds.